src/lib.rs

```diff
@@ -433,6 +476,11 @@ pub enum Op {
     ///
     /// {A, B, C} - Tuple(3) - {D(A, B, C)}
     Tuple(usize),
+    /// Creates a new [List] with the given size and place it on the top
+    /// of the stack.
+    ///
+    /// {A, B, C} - List(3) - {D(A, B, C)}
+    List(usize),
```

Comment:
I'm starting to wonder if this and Op::Tuple should be functions. Feels wierd to specify tuples here. But then again we probably get a speed increase.

Reply:
I don't think we'll run into problems with "too many ops" any time soon.